typescript/sdk/src/core/HyperlaneCoreDeployer.ts

```diff
@@ -106,7 +106,11 @@ export class HyperlaneCoreDeployer extends HyperlaneDeployer<
         ),
       );
     } catch (e: any) {
-      if (!e.message.includes('already initialized')) {
+      if (
+        !e.message.includes('already initialized') &&
+        // Some RPC providers dont return the revert reason (nor allow ethers to parse it), so we have to check the message
+        !e.message.includes('Reverted 0x08c379a')
```

Comment:
Is this an error message specific to core deployments or something we would want to catch for all providers?

Reply:
Just core deployments